dbms/src/Functions/greatCircleDistance.cpp

```diff
@@ -7,77 +7,87 @@
 #include <Functions/FunctionHelpers.h>
 #include <Functions/FunctionFactory.h>
 #include <ext/range.h>
-#include <cmath>
+#include <math.h>
```

Comment:
Why we want to use C library instead of C++?

Reply:
My KDevelop IDE failed to highlight float versions of math functions and I changed the header to math.h (but probably this change is irrelevant).

Reply:
Yes, it's irrelevant.